src/ReviewForm.test.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import ReviewForm from './ReviewForm';
+
+describe('ReviewForm', () => {
+  const handleSubmit = jest.fn();
+  const handleChange = jest.fn();
+
+  const renderReviewForm = (loginFields = {}) => render(
+    <ReviewForm
+      onSubmit={handleSubmit}
+      onChange={handleChange}
+      fields={loginFields}
+    />,
+  );
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('input-controls', () => {
```

Comment:
```suggestion
  describe('input controls', () => {
```
케밥 케이스를 사용하기보다는 그냥 띄어쓰기로 작성해주는건 어떤가요?